src/App.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import MainContainer from './containers/MainContainer';
+
+class App extends Component {
```

Comment:
Hi Dave, 

Can we refactor this into a functional component? This is ok but the legacy way to write react. 

import styled from 'styled-components';
import React from 'react';
const App = () => (
  <div>
  </div>
);
export default App;